resources/shaders/dxr_pathtracer_functions.hlsl

```diff
@@ -136,9 +167,17 @@ float3 ggxIndirect(float3 hit_pos, float3 fN, float3 N, float3 V, float3 albedo,
 		// ##### BOUNCE #####
 		nextRand(seed);
 		const float3 rand_dir = getCosHemisphereSample(seed, N);
-		float3 irradiance = TraceColorRay(hit_pos + (EPSILON * N), rand_dir, depth, seed);
-		if (dot(N, rand_dir) <= 0.0f) irradiance = float3(0, 0, 0);
+		float3 irradiance = 0;
+		if (use_raycone)
+		{
+			TraceColorRayCone(hit_pos + (EPSILON * N), rand_dir, depth, seed, cone);
+		}
+		else
+		{
+			TraceColorRay(hit_pos + (EPSILON * N), rand_dir, depth, seed);
+		}
 
+		if (dot(N, rand_dir) <= 0.0f) irradiance = float3(0, 0, 0);
```

Comment:
Perhaps split this up into multiple lines? The statement right after the check could potentially be a place for future bugs due to the lack of "{" and "}".

Reply:
not new code.

Reply:
So it's not worth resolving? If not, feel free to commit the file the way it is. :)